components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -85,6 +85,9 @@ export interface GitpodServer extends JsonRpcServer<GitpodClient>, AdminServer,
     getLoggedInUser(): Promise<User>;
     getTerms(): Promise<Terms>;
     updateLoggedInUser(user: Partial<User>): Promise<User>;
+    needsVerification(): Promise<boolean>;
```

Comment:
Do we need the method `needsVerification()`? I don't see it being used in the frontend (anymore). Feels like a leftover from an ealier version?